fiftyone/utils/eval/coco.py

```diff
@@ -22,11 +36,13 @@ class COCOEvaluationConfig(DetectionEvaluationConfig):
         classwise (None): whether to only match objects with the same class
             label (True) or allow matches between classes (False)
         iscrowd ("iscrowd"): the name of the crowd attribute
+        iou_threshs([0.5::0.05::0.95]): 10 IoU thresholds used to compute mAP
     """
 
     def __init__(self, iscrowd="iscrowd", **kwargs):
         super().__init__(**kwargs)
         self.iscrowd = iscrowd
+        self.iou_threshs = [x / 100 for x in range(50, 100, 5)]
```

Comment:
you need to parse `iou_threshs` from `__init__()` in case the user provided a customized value